From e1022216d9477309fc3ec51637ff1f59eb298530 Mon Sep 17 00:00:00 2001
From: Adam Fontenot <adam.m.fontenot@gmail.com>
Date: Fri, 12 Apr 2024 01:16:06 -0400
Subject: [PATCH] Fix build issues arising from syntax error in acinclude.m4

The acinclude.m4 file places the closing `fi` in the wrong location,
resulting in builds on some systems failing, e.g. Arch Linux:
https://aur.archlinux.org/packages/celestia

The `fi` belongs where it is placed by this commit because there is
no `if` inside the `AC_CACHE_VAL`. Rather, the statement is intended
to close the conditional `if test -n "$LIBJPEG"`.
---
 acinclude.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/acinclude.m4 b/acinclude.m4
index 49a086f893..1fe80ef680 100644
--- a/acinclude.m4
+++ b/acinclude.m4
@@ -2630,8 +2630,8 @@ AC_TRY_LINK(
 LIBS="$ac_save_LIBS"
 CXXFLAGS="$ac_save_CXXFLAGS"
 AC_LANG_RESTORE
-fi
 ])
+fi
 
 if eval "test ! \"`echo $kde_cv_qt_jpeg`\" = no"; then
   AC_MSG_RESULT(yes)
